_posts/2024-04-24-ASUworkshop.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Save the Date - in-person workshop at Arizona State University
+---
+
+<img
+src="/public/images/ASUworkshop_cactus.png"
+title="MDAnalysis ASU workshop 2024" alt="MDAnalysis Workshop Just 25-26th Arizona State University"
```

Comment:
```suggestion
title="MDAnalysis ASU workshop 2024" alt="MDAnalysis Workshop June 25-26th Arizona State University"
```

Might be a typo, noticed while forwarding to one of my postdocs

Reply:
thanks, fixed it